define-helpers/define-helpers.js

```diff
@@ -41,7 +41,12 @@ var defineHelpers = {
 		// next if it's already on this instances
 		var instanceDefines = map._instanceDefinitions;
 		if(!instanceDefines) {
-			instanceDefines = map._instanceDefinitions = {};
+			Object.defineProperty(map, "_instanceDefinitions", {
```

Comment:
I don't remember the details now, but some stuff was getting screwed up because `_instanceDefinitions` was included in `.forEach()` loops.